system-int-test/test/cmr/system_int_test/ingest/bulk_update/bulk_update_test.clj

```diff
@@ -860,6 +864,36 @@
       ;; revision-id not changed, format not changed, it's identical to the original-concepts.
       (is (= new-concepts original-concepts)))) 
 
+(deftest bulk-update-update-all-tomb-stone-test
```

Comment:
nit: `tombstone` instead of `tomb-stone`.

Reply:
I'm confused. bulkupdate needs be separated, but tomb-stone need to be combined? Done.